src/Sql/dbo/Views/ProviderOrganizationOrganizationDetailsView.sql

```diff
@@ -10,6 +10,7 @@ SELECT
     PO.[CreationDate],
     PO.[RevisionDate],
     (SELECT COUNT(1) FROM [dbo].[OrganizationUser] OU WHERE OU.OrganizationId = PO.OrganizationId AND OU.Status = 2) UserCount,
+    (SELECT COUNT(1) FROM [dbo].[OrganizationUser] OU WHERE OU.OrganizationId = PO.OrganizationId AND OU.Status >= 0) AssignedSeats,
```

Comment:
❓ What's "status greater than zero" mean? I don't know anything about this column but feels like a strange check to me ... does it go below zero?

Reply:
@withinfocus Yeah that column maps directly to this `server` enum: https://github.com/bitwarden/server/blob/main/src/Core/AdminConsole/Enums/OrganizationUserStatusType.cs

The new column should pull the count of any organization user that's Invited, Accepted or Confirmed rather than just Confirmed which is what the `UserCount` column does.